ci/scripts/e2e-sink-test.sh

```diff
@@ -95,6 +95,7 @@ risedev slt './e2e_test/sink/file_sink.slt'
 risedev slt './e2e_test/sink/license.slt'
 risedev slt './e2e_test/sink/rate_limit.slt'
 risedev slt './e2e_test/sink/auto_schema_change.slt'
+risedev slt 'e2e_test/sink/kafka/test_bytes_format.slt'
```

Comment:
Maybe this shall be added here instead (and I am wondering whether we can make it a wildcard - not required in this PR):
https://github.com/risingwavelabs/risingwave/blob/7692aa89c56c0236613b4d57e3d3da28170837a9/ci/scripts/e2e-kafka-sink-test.sh#L149-L163